tools/cldr-code/src/main/java/org/unicode/cldr/tool/CompareEn.java

```diff
@@ -112,12 +95,62 @@ private static void uplevelEnGB() throws IOException {
         }
     }
 
+    static final class Values {
+        String valueGBR;
+        String value001R;
+    }
+
+    static enum FilterStatus {
+        /** don't make the change */
+        skip,
+        /** don't make the change, but note in the comparison file */
+        skipButNote,
+        /** make the change */
+        accept
+    }
+
+    /**
+     * Check whether we should skip or not. If we accept the change, then values contains the values.
+     * @param path
+     * @param en_GBR
+     * @param en_001R
+     * @param values
+     * @return
+     */
+    public static FilterStatus failsFilter(String path, CLDRFile en_GBR, CLDRFile en_001R, Values values) {
+        if (path.startsWith("//ldml/identity")) {
+            return FilterStatus.skip;
+        }
+        if (path.startsWith("//ldml/dates/timeZoneNames/") && path.contains("/short/")) {
+            if (VERBOSE) {
+                System.out.println("Skipping\t" + path);
+            }
+            return FilterStatus.skipButNote;
+        }
```

Comment:
Should add a skip for standard time formats

Reply:
done (also datetimeSkeleton)